config/ext/rspec.yml

```diff
@@ -278,9 +279,10 @@ RSpec/DescribedClass:
   SupportedStyles:
     - described_class
     - explicit
+  OnlyStaticConstants: false
```

Comment:
Habe nicht ganz verstanden worum es hier geht. 
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#onlystaticconstants-true-default

Reply:
I'm not 100% sure either but I to my understading, with this setting set to false, you cannot write
```
describe MyClass do
  subject { MyClass::CONSTANT }
end
```

but you have to write
```
describe MyClass do
  subject { described_class::CONSTANT }
end
```
instead

Reply:
I think we've had a discussion on this somewhere. Let's not enforce this, thus disable the cop.